packages/account/src/Types/common-prop.types.ts

```diff
@@ -0,0 +1,9 @@
+export type TReactChangeEvent = React.ChangeEvent<HTMLInputElement>;
+
+export type TReactChildren = React.ReactNode;
+
+export type TReactMouseEvent = React.MouseEvent<HTMLElement>;
+
+export type TReactFormEvent = React.FormEvent<HTMLInputElement>;
+
+export type TReactElement = React.ReactElement;
```

Comment:
This will create an unnecessary abstraction in the code, I suggest we keep `React.x` to make sure that the types are default React types.

Reply:
Will use this file to hold commonly used types like api data etc